hugo/layouts/partials/header.html

```diff
@@ -1,7 +1,8 @@
 {{ $ctx := .ctx }}
 {{ $categories := $ctx.Site.Data.categories.categories.contents }}
 {{ $project := $ctx.Site.Data.project }}
-{{ $category := default "" $ctx.Site.Data.project.category }}
+{{ $isDocSite := default false $project.is_doc_site }}
```

Comment:
if `$ctx.Site.Data.project` is set, then the site is a documentation site, isn't it?